_posts/: 2020-06-05-Vallie-Joseph.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "A new hobby or skill I've picked up during lockdown..."
+author: Vallie
+date: 2020-06-03
+---
+
+![Image of 3D Printer](https://media.giphy.com/media/R3H7UiNMdykYE/giphy.gif)
+
+Comment your reply here.
```

Comment:
Q replied "Can I bother you to make me stuff? :)"

Reply:
$$